src/AssetManager.php

```diff
@@ -133,8 +133,6 @@ public function assets(): array
             $found[$class][$handle] = $class;
```

Comment:
sry for asking again, but shouldn't we assign here the `$asset` instead of `$class` as value? The `$class` is already present as key. :)

Reply:
No need to be sorry when something is wrong :)